bin/transform_local

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
```

Comment:
Please name `transform-local`, for consistency with Transformer PSC post-refactor.